core/crypto/vrf/p256/p256_test.go

```diff
@@ -187,35 +183,26 @@ func TestVRF(t *testing.T) {
 // Test vectors in core/testdata are generated by running
 // go generate ./core/testdata
 func TestProofToHash(t *testing.T) {
-	directoryFile := "../../../testdata/directory_get_and_update.json"
-	f, err := os.Open(directoryFile)
+	transcript, err := testdata.ReadTranscript("TestEmptyGetAndUpdate")
 	if err != nil {
-		t.Fatalf("ReadFile(%v): %v", directoryFile, err)
+		t.Fatal(err)
 	}
-	defer f.Close()
-	var directory pb.Directory
-	if err := jsonpb.Unmarshal(f, &directory); err != nil {
-		t.Fatalf("jsonpb.Unmarshal(): %v", err)
-	}
-	pk, err := NewVRFVerifierFromRawKey(directory.GetVrf().GetDer())
+
+	pk, err := NewVRFVerifierFromRawKey(transcript.GetDirectory().GetVrf().GetDer())
 	if err != nil {
 		t.Fatalf("NewVRFVerifier failure: %v", err)
 	}
 
-	respFile := "../../../testdata/get_and_update.json"
-	b, err := ioutil.ReadFile(respFile)
-	if err != nil {
-		t.Fatalf("ReadFile(%v): %v", respFile, err)
-	}
-	var getUserResponses []testdata.ResponseVector
-	if err := json.Unmarshal(b, &getUserResponses); err != nil {
-		t.Fatalf("Unmarshal(): %v", err)
-	}
-	for _, tc := range getUserResponses {
-		t.Run(tc.Desc, func(t *testing.T) {
-			_, err := pk.ProofToHash([]byte(tc.UserID), tc.GetUserResp.GetLeaf().GetVrfProof())
-			if err != nil {
-				t.Errorf("ProofToHash(%v): %v)", tc.Desc, err)
+	for _, rpc := range transcript.Rpcs {
+		t.Run(rpc.Desc, func(t *testing.T) {
+			switch pair := rpc.ReqRespPair.(type) {
+			case *tpb.Unary_GetUser:
```

Comment:
Does it need a `default: t.Fatalf` case, or it skips non-`GetUser` requests?

Reply:
It will skip the other requests, but I'll add a default case just to make sure we don't surprise ourselves.